src/Datadog.Trace.ClrProfiler.Managed/Integrations/AWS/AwsSdkTags.cs

```diff
@@ -0,0 +1,23 @@
+namespace Datadog.Trace.ClrProfiler.Integrations
+{
+    internal static class AwsSdkTags
```

Comment:
Would you be opposed to renaming the class `AmazonSdkTags`? I know the tags in this class all start with `aws` but it seems a little confusing that we have different prefixes for filenames in this directory.

Reply:
Maybe use "aws" for all of them instead of "amazon"?

Reply:
"aws" across the board makes sense to me, I'd support that change 👍

Reply:
I'm fine with that, the services in Amazon are listed as `Amazon.Sqs`, `Amazon.DynamoDB`, etc.
Amazon just... spoke nicely for me. Rather than seeing a giant acronym for everything.
Either way, thoughts?

Reply:
Amazon has a better ring to it, but I feel like we'll always be referring to these integrations as `aws` like "Hey we gotta add more to that AWS integration". So my preference would be to name things `aws` as much as possible. Like change everything from `Amazon` to `aws` and then if something sounds ridiculous we can roll it back